paper/paper.md

```diff
@@ -0,0 +1,141 @@
+energetic this coupling can break down, and we have two different
+implementations based on spectral deferred corrections (SDC), a method
+that aims prevent the hydro and reactions from becoming decoupled.  The
+simplified SDC method uses the CTU PPM hydro together with an
+iterative scheme to fully couple the reactions and hydro, still to
+second order [@simple_sdc].  Both of these methods have a retry
+scheme, where a timestep will be rejected if the burning solve fails
+to meet its tolerance, negative densities are generated, or we violate
+one of the timestepping criterion.  Alternately, we have implemented a
+traditional SDC method that couples hydro and reactions to both second
+and fourth-order in space and time [@castro_sdc] (at present, this
+method is single-level only).
+
+In addition to reactive hydrodynamics, Castro includes full
+self-gravity with isolated boundary conditions and rotation, both
+implemented in an energy-conserving fashion, explicit thermal
+diffusion, and gray [@castroII] and multigroup [@castroIII] flux
+limited diffusion radiation hydrodynamics.  A constrained transport
+MHD solver based on the CTU algrothm is also available, and can use
+the same physics sources.  Castro can use an arbitrary equation of
+state and reaction network and these microphysics routines are
+provided by the StarKiller project [@starkiller].
+
+Castro is built on the AMReX [@AMReX] adaptive mesh refinement (AMR)
+library and is largely written in C++ with a few Fortran compute
+kernels.  AMR levels are advanced at their own timestep (subcycling)
+and jumps by factors of 2 and 4 are supported between levels.  We use
+MPI to distribute AMR grids across nodes and using logical tiling with
+OpenMP to divide a grid across threads for manycore machines or CUDA
+to spread the work across GPU threads on GPU-based machines.  All of
+the core physics can run on GPUs (CTU hydro, CT-CTU MHD, self-gravity,
+diffusion, reactions) and has been shown to scale well to 1000s of
+GPUs [@castro_2019] and 100,000s of CPUs [@castro_2017].  For
+performance portability, the same compute kernel is used for CPUs or
+GPUs, using lambda-capturing for C++ kernels.
+
+While there are a number of astrophysical simulation codes, Castro
+offers a few unique features.  The original motivation for developing
+Castro was to build a simulation code based on a modern,
+well-supported AMR library (BoxLib which evolved to AMReX), using
+unsplit integration techniques and targeting problems in nuclear
+astrophysics.  The radiation solver was a key design consideration in
+the early development.  The large number of application codes using
+AMReX means that Castro continually gains optimizations for new
+architectures.  As Castro evolved, we adopted a fully open development
+model (as does the Enzo [@enzo] code, for example).  We pride ourselfs in
+making all of the science problems available Castro git repository as
```

Comment:
```suggestion
making all of the science problems available in the Castro git repository as
```

Reply:
I think this still is missing a couple of words?

Reply:
fixed